### PR TITLE
Improve incident

### PR DESCRIPTION
**Pull request by rpiazza**
improve incident and handling of stix1 statement type

### CODE REVIEW

File: docs/stix-mappings.rst [hunk 1]
```diff
@@ -37,7 +37,7 @@ objects - but their mapping can be found in the :ref:`cyber_observables` section
 +-----------------------------+----------------------------+
 | ``et:Configuration``        | *not converted*            |
 +-----------------------------+----------------------------+
-| ``Incident``                | *not converted by default* |
+| ``Incident``                | ``Incident`` * in 2.1*     |
```
Comment (emmanvg) [inline docs/stix-mappings.rst:40]:
```suggestion
| ``Incident``                | ``incident`` *in 2.1*      |
```

File: idioms-json-2.1-extensions/incident-malware.json [hunk 2]
```diff
@@ -22,14 +22,35 @@
...
+                        "Antivirus"
                     ],
                     "extension_type": "property-extension",
+                    "intended_effects": [
+                        {
+                            "Degradation of Service": {
+                                "description": "Bring down server"
+                            }
+                        }
+                    ],
```
Comment (emmanvg) [inline idioms-json-2.1-extensions/incident-malware.json:39]: Something looks weird about this one.... deeply nested structure. Would in it be:

``` json
{
    "value": "Degradation of Service",
    "description": "Bring down server"
}
```

File: idioms-json-2.1-custom/incident-malware.json [hunk 3]
```diff
@@ -3,29 +3,49 @@
...
+                    "source_name": "JIRA"
+                }
+            ],
             "id": "incident--1b75ee8f-44d6-819a-d729-09ab52c91fdb",
+            "intended_effects": [
+                {
+                    "description": "Bring down server",
+                    "value": "Degradation of Service"
+                }
+            ],
```
Comment (emmanvg) [inline idioms-json-2.1-custom/incident-malware.json:42]: If these are the custom test results, should these be prefixed with `x_elevator_<property_name>` (happens in many areas)

**Comment by emmanvg** [comment]
Not clear what areas of code need to be changed, but findings are mainly on content.

File: docs/stix-mappings.rst (hunk 1 above)
Comment (rpiazza) [reply docs/stix-mappings.rst:40]: done

File: idioms-json-2.1-custom/incident-malware.json (hunk 3 above)
Comment (rpiazza) [reply idioms-json-2.1-custom/incident-malware.json:42]: I had this idea that "Incident" should use the "x_" prefix, but for the life of me I can't remember my reasoning - so I took it out

File: idioms-json-2.1-extensions/incident-malware.json (hunk 2 above)
Comment (rpiazza) [reply idioms-json-2.1-extensions/incident-malware.json:39]: I took care of this

**Review by emmanvg — APPROVED** [review]
Looks good

File: docs/stix-mappings.rst [hunk 4]
```diff
@@ -37,7 +37,7 @@ objects - but their mapping can be found in the :ref:`cyber_observables` section
 +-----------------------------+----------------------------+
 | ``et:Configuration``        | *not converted*            |
 +-----------------------------+----------------------------+
-| ``Incident``                | *not converted by default* |
+| ``Incident``                | ``Incident`` *in 2.1*     |
```
Comment (emmanvg) [inline docs/stix-mappings.rst:40]:
```suggestion
| ``Incident``                | ``incident`` *in 2.1*      |
```

**Pull request merged**